src/Aspire.Dashboard/Components/Pages/ConsoleLogs.razor

```diff
@@ -7,11 +7,11 @@
 <div class="resource-logs-layout">
     <h1>Console Logs</h1>
     <FluentToolbar Orientation="Orientation.Horizontal">
-        <FluentSelect @ref="_resourceSelectComponent" TOption="ResourceViewModel"
+        <FluentSelect @ref="_resourceSelectComponent" TOption="Option<string>"
```

Comment:
That'll allow us to add icons easily too if we decide to do so.

Reply:
How long has `Option<T>` been in FluentUI? I added our own version - `SelectViewModel<T>` - because I never noticed it.

Reply:
Hmm, why does `Option<T>` have Text and Value properties both of `T`? Surely Text should always be a string...

Reply:
> How long has `Option<T>` been in FluentUI? I added our own version - `SelectViewModel<T>` - because I never noticed it.

It's been there since August I guess. I only really noticed it when I looked into how to add icons and it looks like using `Option<T>` is the only way to do so:

https://github.com/microsoft/fluentui-blazor/blob/41bc71d065f0cd7f0006345cc5a6be101e1dff17/src/Core/Components/List/ListComponentBase.cs#L418C25-L437C26


> Hmm, why does `Option<T>` have Text and Value properties both of `T`? Surely Text should always be a string...

Yeah I don't know, it seems off. Seems like `Value` should be `T` and `Text` should be `string`. But there's no special handling for `Option<T>` (other than Icon) so you still have to use the `OptionText` and `OptionValue` parameters to set the display text and value.